apps/web/package.json

```diff
@@ -105,6 +105,7 @@
     "next-seo": "^6.0.0",
     "next-themes": "^0.2.0",
     "nodemailer": "^6.7.8",
+    "nuqs": "^1.20.0",
```

Comment:
https://nuqs.47ng.com/ 

We've talked about using this a lot before so thought now would be a good time